README.md

```diff
@@ -147,4 +145,42 @@ with f:
     f.search(doc_query, parameters={'filter': filter_})
 ```
 
+### Limit returning results
+
+In some cases, you will want to limit the total number of retrieved results. `QdrantIndexer` uses the `limit` argument 
+from the `match` function to set this limit. Note that when using `shards=N`, the `limit=K` is the number of retrieved results for **each shard** and total number of retrieved results is `N*K`. By default, `limits` is set to `20`. For more information about shards, please read [Jina Documentation](https://docs.jina.ai/fundamentals/flow/topology/#partition-data-by-using-shards)
+
+```python
+f =  Flow().add(
+    uses='jinahub+docker://QdrantIndexer',
+    uses_with={'match_args': {'limit': 10}})
+```
+
+### Configure the other search behaviors
+
+You can use `match_args` argument to pass arguments to the `match` function as below.
+
+```python
+f =  Flow().add(
+     uses='jinahub+docker://QdrantIndexer',
+     uses_with={
+         'match_args': {
+             'metric': 'euclidean',
+             'use_scipy': True}})
+```
+
+- For more details about overriding configurations, please refer to [here](https://docs.jina.ai/fundamentals/executor/executor-in-flow/#special-executor-attributes).
+- You can find more about the `match` function at [here](https://docarray.jina.ai/api/docarray.array.mixins.match/#docarray.array.mixins.match.MatchMixin.match)
```

Comment:
I think it's better to refer to qdrant's find method, because the in-memory matching is not used in qdrant